src/ring_theory/power_series/basic.lean

```diff
@@ -1178,6 +1201,21 @@ lemma mul_inv_of_unit (φ : power_series R) (u : units R) (h : constant_coeff R
   φ * inv_of_unit φ u = 1 :=
 mv_power_series.mul_inv_of_unit φ u $ h
 
+/-- Two ways of removing the constant coefficient of a power series are the same. -/
+lemma sub_const_eq_shift_mul_X (φ : power_series R) :
+  φ - C R (constant_coeff R φ) = power_series.mk (λ p, coeff R (p + 1) φ) * X :=
+begin
+  rw [sub_eq_iff_eq_add],
+  exact eq_shift_mul_X_add_const φ,
+end
```

Comment:
This should golf to something like:
```suggestion
sub_eq_iff_eq_add.mpr $ eq_shift_mul_X_add_const φ
```
although I haven't tried

Reply:
done